default_python/requirements-dev.txt

```diff
@@ -6,6 +6,9 @@
 ## pytest is the default package used for testing
 pytest
 
+## Add code completion support for DLT
+databricks-dlt
```

Comment:
Unfortunately, this package is really not useful yet without return types. We can already add it but we probably shouldn't enable linting for DLT-enabled projects until a fix is released.

Reply:
Yeah, the missing return types are a problem. At least this gets rid of the squiggly lines